src/vimish/index.js

```diff
@@ -322,15 +323,22 @@ exports.stream = exports.streams = Command.of(
     }
 
     Streams.Settings.set(`active.${stream}`, state)
-    Bus.emit(Bus.events.FLASH, {
-      kind: "ok",
-      message: `${stream} stream is now ${
-        state == 1 ? "on" : "off"
-      }`,
-    })
+    Bus.emit(Bus.events.REDRAW)
   }
 )
 
+// `:theme dark-king`
+exports.theme = Command.of(["value"], async ({ value }) => {
+  if (value in THEME_NAMES) {
```

Comment:
This should be a return early/error first pattern.

basically anytime you write an if/else statement and you have a tiny "dangling else" it should be refactored to be:

```javascript
if (!(value in THEME_NAMES)) {
  // handle dangling case first
}
```
This prevents complexity from building up over time as more cases are added, and makes it easier to verify what is required for the `Ok` path to be run.

Here is a pretty solid overview of else-less programming paradigm if you're interested
https://blog.timoxley.com/post/47041269194/avoid-else-return-early

Reply:
Can-do.